Installers/default.build

```diff
@@ -34,6 +34,17 @@
         <exec program="makensis" workingdir="Windows" basedir="C:\Program Files (x86)\NSIS">
 		      <arg value="MonoGame.nsi"/>
         </exec>
+        <echo append="false" file="Windows/hash.bat">
+@ECHO OFF
+FOR /f "tokens=*" %%i IN ('@certutil -hashfile %1 %2 ^| find /v "hash of file" ^| find /v "CertUtil"') DO SET r=%%i
+SET r=%r: =%
+ECHO %r%  %1 > %3
+        </echo>
```

Comment:
Seems like a bit of a dirty trick to do this here.  Why not just create a bat file and add it to the repo?

Reply:
I can no problem.